source/demo-code/recipe-info-rough/recipePage.html

```diff
@@ -128,5 +129,19 @@
             </div>
         </section>
     </main>
+    <div class="timer">
+        <div id="timeInput" class="timeInput">
+            <input type="number" name ="hours" id ="hours" placeholder="Hours" min="0" max="23" size = "5" maxlength="2" value="0">
+            <input type="number" name="minutes" id="minutes" placeholder="Minutes" min="0" max="59" size = "5" maxlength="2" value="0">
+            <input type="number" name="seconds" id="seconds" placeholder="Seconds" min="0" max="59" size = "5" maxlength="2" value="0">
+            <button href="javascript:void(0);" class="setTime" id="setTime" onclick="setTimer()">Set Timer</button>
+        </div>
+        
+        <p id="timer">Begin Timer</p>
+        <button id="pause" onclick="pauseTimer()">Pause</button>
+        <button id="reset" onclick="resetTimer()">Reset</button>
+        <button id="shown" onclick="displayTimer()">Minimize</button>
+        <audio class="hidden" src=""></audio>
```

Comment:
```
var audioElement = new Audio('sound.mp3');
audioElement.play()
```
Might be easier to directly play audio from JavaScript when the timer finishes and we don't need to create an element.

Reply:
Changed